config.json.example

```diff
@@ -18,6 +18,16 @@
             1000,
             2000
         ],
+        "ignorePrefix": [
+            "!", 
+            "m!", 
+            "m.", 
+            "p!", 
+            "//",
+            "i-",
+            ",",
+            "git"
```

Comment:
The example config should not have any defined values in beyond one or two examples.